components/bl-autocomplete-component/src/index.js

```diff
@@ -0,0 +1,56 @@
+import { useCallback, useRef } from 'react';
+
+import { Suggestions } from './components/suggestions';
+import { TextField } from './components/text-field';
+import { actions } from './hooks/useAutocomplete';
+import { useAutocomplete } from './hooks/useAutocomplete';
```

Comment:
if you have hooks dir, create please index and reexport it.

That helps to make here 

```
import { useAutocompleteClassList, useOnClickOutside} from './hooks';
```